### PR TITLE
[ADD] Module to add EORI number to company form

### PR DESCRIPTION
**Pull request by lfreeke**

### CODE REVIEW

**Comment by luc-demeyer** [comment]
Thanks for this contribution.
Can you fix the rst error ?
I am not sure if the 'intrastat' repo is the best place to host this module since EORI is for 'extracom' business whereas intrastat is for 'intracom' but we already host the 'product_harmonized_system' module  so we can imho merge this module into this repo and think about a more generic import/export/customs kind or repo for the upcoming Odoo releases.

**Comment by pedrobaeza** [comment]
We can also rename the repository if needed.

**Comment by lfreeke** [comment]
@luc-demeyer I fixed the rst errors. It seems Travis is failing because of another module. 

I had trouble to find a place for this module, it doesn't really fit in other repo's too. A Custom repo is a good idea, we all have more or less the same requirements in Europe.

**Comment by luc-demeyer** [comment]
I'll fix the travis error via a separate PR.
What about renaming the repo to intrastat-extrastat ?

**Comment by pedrobaeza** [comment]
OK, seems reasonable

**Comment by luc-demeyer** [comment]
@alexis-via do you agree with rename of repo to  intrastat-extrastat ?

**Comment by alexis-via** [comment]
I must say I don't really like the idea of renaming the project.
Let's have this module for EORI here and keep the project with it's current name. Let's keep things simple!

**Comment by pedrobaeza** [comment]
@alexis-via there's no problem in renaming. References to old name will still work without problems. We have already done it for other repositories when the content has been adjusted.

**Comment by alexis-via** [comment]
Yes, I know renaming the project is technically not a problem. But I think that the current name of the project should be kept because the word "intrastat" has been used in Odoo for a very long time (at least since v5, cf https://github.com/odoo/odoo/tree/5.0/addons/report_intrastat), so everybody knows what it means ! If we change the name, it will be another story !!!
And I think we can accept this module without changing the name of the project.

**Comment by pedrobaeza** [comment]
But the word will be kept. We are only adding -extrastat...

**Comment by alexis-via** [comment]
OK, fine :) :)

File: company_eori_number/README.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+===================
+company_eori_number
```
Comment (daramousk) [inline company_eori_number/README.rst]: Set a proper name for your module here.

File: company_eori_number/README.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+===================
+
+This module was written to extend the functionality of the company form to support EORI number (Economic Operators Registration and Identification number) .
+
+
+Installation
+============
+
+To install this module, you need to:
```
Comment (daramousk) [inline company_eori_number/README.rst]: Remove this or give proper instructions.

File: company_eori_number/README.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+
+#. Install the module.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. No need for configuration
```
Comment (daramousk) [inline company_eori_number/README.rst]: If something does not apply, remove the section completely (ex. remove Configuration)

File: company_eori_number/README.rst [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+To use this module, you need to:
+
+#. After installation of the module a new field 'EORI number' will be added to the company form.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+    :alt: Try me on Runbot
+    :target: https://runbot.odoo-community.org/runbot//8.0
+
+
+Known issues / Roadmap
```
Comment (daramousk) [inline company_eori_number/README.rst]: Same here, this can be removed

File: company_eori_number/README.rst [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+=============
+
+To configure this module, you need to:
+
+#. No need for configuration
+
+Usage
+=====
+
+To use this module, you need to:
```
Comment (daramousk) [inline company_eori_number/README.rst]: You could fill this in.

File: company_eori_number/README.rst [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* lfreeke <lfreeke@therp.nl>
```
Comment (daramousk) [inline company_eori_number/README.rst]: Set your name here before your email

File: company_eori_number/__openerp__.py [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "company_eori_number",
+    "version": "8.0.1.0.0",
+    "author": "Therp BV,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "",
```
Comment (daramousk) [inline company_eori_number/__openerp__.py]: Fill these in.

File: company_eori_number/models/res_company.py [hunk 8]
```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp import fields,  models
```
Comment (daramousk) [inline company_eori_number/models/res_company.py]: Multiple spaces between fields and models.

Comment (daramousk) [reply company_eori_number/models/res_company.py]: Add an extra line here.

**Comment by luc-demeyer** [comment]
@lfreeke @daramousk 
I have updated the PR with the feedback of daramousk. Please review.

@pedrobaeza 
can you rename the repo to 'intrastat-extrastat' ?

**Review by daramousk — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@luc-demeyer, I think @alexis-via is still not convinced about the name change.

**Comment by luc-demeyer** [comment]
3 days ago Alexis reacted with "OK, fine :) :)" (cf. conversation history before the comments of daramousk) hence you can go ahead.

**Comment by pedrobaeza** [comment]
Repository renamed.

**Review by lfreeke — COMMENTED** [review]
:+1:

**Comment by sbidoul** [comment]
@pedrobaeza when renaming repositories, one needs to think about weblate.

**Comment by pedrobaeza** [comment]
OK, let me try. Maybe we should make a Wiki page about renaming?

**Comment by pedrobaeza** [comment]
OK, as both were duplicated right now, but with exact contents, I have removed the projects with old name.

**Pull request merged**